### PR TITLE
NFT.py all methods commented

### PR DESCRIPTION
**Pull request by PrasoonPratham**

### CODE REVIEW

File: thirdweb/modules/nft.py [hunk 1]
```diff
@@ -256,8 +290,10 @@ def set_restricted_transfer(self, restricted: bool = False):
         """
 
         tx = self.__abi_module.set_restricted_transfer.build_transaction(
-            restricted, self.get_transact_opts())
+            restricted, self.get_transact_opts()
+        )
         self.execute_tx(tx)
 
     def get_abi_module(self) -> NFT:
+        """To be removed"""
```
Comment (atbe) [inline thirdweb/modules/nft.py:298]: Lets remove this comment, these methods need to stay for the module to function correctly.

Comment (PrasoonPratham) [reply thirdweb/modules/nft.py]: Makes sense, changes made.

File: thirdweb/modules/nft.py [hunk 2]
```diff
@@ -12,28 +12,42 @@
...
         self.__abi_module = NFT(client, address)
+        """
+        The ABI makes calls the EVM. Client is default 'Web3' and Address is nft module contract address.
+        """
 
     def mint(self, arg: MintArg) -> NftType:
         """
-        Mints a new token
+        - Mints a new token. 
+        - Arguments passed: Note, a class is used -> MintArg(name, description, image_uri, properties) *Preferrably, using a link
```
Comment (ayshptk) [inline thirdweb/modules/nft.py:41]: can we remove ` *Preferrably, using a link`?

File: thirdweb/modules/nft.py [hunk 3]
```diff
@@ -12,28 +12,42 @@
...
+        - Returns the 'NftMetadata(name,description,image,properties,id,uri)
```
Comment (ayshptk) [inline thirdweb/modules/nft.py:42]: this should either be \`NftMetadata(name,description,image,properties,id,uri)\` or NftMetadata(name,description,image,properties,id,uri)

File: thirdweb/modules/nft.py (hunk 2 above)
Comment (ayshptk) [reply thirdweb/modules/nft.py:41]: should probably be changed to the other format like in [this one](https://github.com/nftlabs/nftlabs-sdk-python/pull/15/files#diff-85ff59513ad491407f1f7bcaad29cf1d428e525190145629436e39407893033dR222) or the other way round. I like the second one more

File: thirdweb/modules/nft.py [hunk 4]
```diff
@@ -12,28 +12,42 @@
...
     def mint_to(
-            self,
-            to_address: str,
-            arg: MintArg,
+        self,
+        to_address: str,
+        arg: MintArg,
     ) -> NftType:
         """
         Mints a new token
```
Comment (ayshptk) [inline thirdweb/modules/nft.py:53]: we should probably maintain consistency here by displaying a format similar to what is written for the [mint method](https://github.com/nftlabs/nftlabs-sdk-python/pull/15/files#diff-85ff59513ad491407f1f7bcaad29cf1d428e525190145629436e39407893033dL27) or mentioning that the usage is same as the `mint` method.

**Review by ayshptk — CHANGES_REQUESTED** [review]

**Review by ayshptk — APPROVED** [review]

**Pull request merged**